src/screens/ResultScreen.tsx

```diff
@@ -124,7 +132,7 @@ const ResultScreen = ({ route, navigation }: ResultScreenPropsType): JSX.Element
 
   const Footer = (
     <>
-      {retryButton}
+      {exercise === ExerciseKeys.vocabularyTrainer && retryButton}
```

Comment:
Why do we only have a retry button for the vocabulary trainer?

Reply:
I guess, the writeExercise is considered to be more complex and therefore has the opportunity to repeat the wrong entries. Maybe in the future, we will add this for singleChoice exercises, too, if it seems necessary.